test/python/tf_unittest_runner.py

```diff
@@ -77,6 +70,16 @@ def all_dirs_to_path(dirname):
 
 
 def list_tests(test_module):
+    """
```

Comment:
Good documentation. Please split the lines such that the width is max 80 columns. Also - reduce the tabs (in commends) to 4 spaces.